hasjob/models/filterset.py

```diff
@@ -76,6 +79,11 @@ def url_for(self, action='view', _external=True, **kwargs):
         kwargs.setdefault('subdomain', self.board.name if self.board.not_root else None)
         return super(Filterset, self).url_for(action, name=self.name, _external=_external, **kwargs)
 
+    @property
+    def proxy(self):
+        """Proxy for setting association proxies in forms. See `FiltersetAssocationsForm`"""
```

Comment:
The comment is incorrect. The proxy is for the use of a `FormField` and has nothing to do with association proxies.